### PR TITLE
Use fasthttp at consumer

### PR DESCRIPTION
**Pull request by jzding**
- Use fasthttp to improve rest api performance at consumer
- Log level was hardcoded to Debug level. Fix this to take level from env
  variable.
- Fix an error for deploy/undelpoy-example in Makefile
- Updated sdk-go version.

Signed-off-by: Jack Ding <jacding@redhat.com>

### CODE REVIEW

File: examples/consumer/main.go [hunk 1]
```diff
@@ -206,41 +206,42 @@ func pingForStatus(resourceID string) {
 	}
 }
 
+func fastHTTPHandler(ctx *fasthttp.RequestCtx) {
+	fmt.Fprintf(ctx, "Hi there! RequestURI is %q", ctx.RequestURI())
```
Comment (aneeshkp) [inline examples/consumer/main.go:210]: remove this is print

Comment (nishant-parekh) [reply examples/consumer/main.go:210]: no not until code freeze on friday..

Comment (jzding) [reply examples/consumer/main.go]: done

File: go.mod [hunk 2]
```diff
@@ -8,9 +8,10 @@ require (
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/prometheus/client_golang v1.11.0
 	github.com/redhat-cne/rest-api v0.1.0
-	github.com/redhat-cne/sdk-go v0.1.0
+	github.com/redhat-cne/sdk-go v0.1.1-0.20210723134838-a808941830c9
```
Comment (nishant-parekh) [inline go.mod:11]: do we need to create a new release of the sdk-go for this or will this work? @aneeshkp

Comment (aneeshkp) [reply go.mod:11]: no yet we will do after release or code freeze

**Comment by aneeshkp** [comment]
@jzding , can this wait till we finish code freeze.

File: examples/consumer/main.go (hunk 1 above)
Comment (jzding) [reply examples/consumer/main.go]: @aneeshkp I think we can include this as part of Redfish Hw Event feature so it is targeted for 4.10 and does not need bug tracking for merging after code freeze.

**Comment by aneeshkp** [comment]
@jzding rebase pls

**Comment by jzding** [comment]
> @jzding rebase pls

rebased

**Review by aneeshkp — APPROVED** [review]

**Pull request merged**